superbench/analyzer/data_diagnosis.py

```diff
@@ -239,10 +244,10 @@ def run(self, raw_data_file, rule_file, baseline_file, output_dir, output_format
             logger.info('DataDiagnosis: Begin to processe {} nodes'.format(len(self._raw_data_df)))
             data_not_accept_df, label_df = self.run_diagnosis_rules(rule_file, baseline_file)
             logger.info('DataDiagnosis: Processed finished')
-            outpout_path = ''
+            output_path = ''
             if output_format == 'excel':
                 output_path = output_dir + '/diagnosis_summary.xlsx'
```

Comment:
could you concatenate path instead of string?

Reply:
updated